src/index.html

```diff
@@ -7,6 +7,43 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="
+      calendar
+      calendar--position
```

Comment:
You add extra modifier and put margin styles there.
But it's still applied to BEM-block, which is incorrect.

Create global block on body tag, make calendar its element using BEM-mix and add margins there.